fep/README.md

```diff
@@ -0,0 +1,26 @@
+# FEP: Fediverse Enhancement Proposals
+
+A [Fediverse Enhancement Proposal](https://codeberg.org/fediverse/fep/src/branch/main/feps/fep-a4ed.md) (FEP) is a document that provides information to the Fediverse community. The goal of a FEP is to improve interoperability and well-being of diverse services, applications and communities that form the Fediverse.
+
+## See also
+
+FEP community links:
+
+- [SocialHub forum category](https://socialhub.activitypub.rocks/c/standards/fep/54)
+- [Current Codeberg repository](https://codeberg.org/fediverse/fep)
+
+FEP-888d is the proposal covering this usage of W3ID:
+
+- [SocialHub forum thread](https://socialhub.activitypub.rocks/t/fep-888d-using-w3id-org-fep-as-a-namespace-for-extension-terms-and-for-fep-documents/3098)
+- [The proposal as currently hosted on Codeberg](https://codeberg.org/fediverse/fep/src/branch/main/fep/888d/fep-888d.md)
+
+## Contact
+
+This prefix was submitted as a PR by a (a@trwnh.com). You may contact them directly for updates via the following methods:
+
+- mailto:a@trwnh.com
+- xmpp:a@trwnh.com
+- https://mastodon.social/@trwnh
+- https://github.com/trwnh/w3id.org/issues
+
+However, it may also / instead be a good idea to field your questions or concerns directly to the SocialHub forum or Codeberg organization, as that is where the community resides. Future PRs may come from other parties, but MUST be accompanied by a relevant community link.
```

Comment:
```suggestion
However, it may be a good idea to also or instead field your questions or concerns directly to the SocialHub forum or Codeberg organization, as that is where the community resides. Future PRs may come from other parties, but MUST be accompanied by a relevant community link.
```

Reply:
Changes incorporated into new squashed commit!